ykrt/src/compile/jitc_yk/trace_builder.rs

```diff
@@ -340,7 +360,9 @@ impl TraceBuilder {
                     let nextinst = blk.insts.last().unwrap();
                     self.handle_idempotent_promote(bid, iidx, val, nextinst)
                 }
-                _ => todo!("{:?}", inst),
+                _ => Err(CompilationError::General(format!(
```

Comment:
I think this should still be `todo`.

Reply:
We discussed this but you may have forgotten. If I make this a `todo` some benchmarks won't run anymore because we are now tracing more code. There's two cases that come up here:
1. `align 1`
2. `no return`
I'm not sure yet what the solution for these are, but I don't think it should go in this PR. But I should look at that sooner rather than later.